File: lua/src/text.lua
--install cjson
--install socket.http
local json = require("cjson")
hc = require('httpclient').new()
--set your own accessKey
postData = json.encode({accessKey="xxxxxxxxxxxxxxxxxxxx", type="ZHIBO", data={tokenId="tokeId_test",text="iphone 7"}})
res = hc:post('http://api.fengkongcloud.com/v2/saas/anti_fraud/text',postData)
print(res["body"])
resJson = json.decode(res)
if resJson["code"] == 1100 then
    if resJson["riskLevel"] == "PASS" then
        --放行
    else if resJson["riskLevel"] == "REVIEW" then
        --人工审核，如果没有审核，就放行
    else if resJson["riskLevel"] == "REJECT" then
        --拒绝
    else
        --异常
    end
end 

File: v2/java/src/main/java/Text.java
import net.sf.json.JSONObject;

import java.util.HashMap;

public class Text {
    private static final String URL = "http://api-text-bj.fengkongcloud.com/v2/saas/anti_fraud/text";
    private static final String ACCESS_KEY = "{ACCESS_KEY}";

    public static void main(String[] args) {
        HashMap<String, Object> payload = new HashMap<String, Object>();
        payload.put("accessKey", ACCESS_KEY);
        payload.put("type", "SOCIAL");
        payload.put("appId", "default");

        HashMap<String, Object> data = new HashMap<String, Object>();
        data.put("text", "{TEXT}");
        data.put("tokenId", "{UID}");

        payload.put("data", data);

        JSONObject json = JSONObject.fromObject(payload);
        JSONObject result = Utils.httpPost(URL, json);

        json = JSONObject.fromObject(result);
		// 通过 code 和 riskLevel 判断返回，具体请参考接口文档
        System.out.println(json.toString());
    }
}